frontend/demo/src/pages/Tags/EmptyStateTags.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+
+import {ReactSVG} from 'react-svg';
+
+import styles from './index.module.scss';
+import {Button} from '@airyhq/components';
+import emptyImage from '@airyhq/components/src/assets/images/empty-state/tags-empty-state.svg';
```

Comment:
cc @bitboxer and @AitorAlgorta: I think we should aim to avoid imports like these: The library's contract are React components, not asset files. If there is no component that wraps this svg, I think we should add it to the demo package directly. What do you think?

Reply:
I agree 100%